examples/two_dimensional_turbulence.jl

```diff
@@ -73,14 +76,19 @@ nothing # hide
 simulation = Simulation(model, Δt=0.1, stop_iteration=0)
 
 anim = @animate for i=1:100
-    simulation.stop_iteration += 10
+    simulation.stop_iteration += 20
     run!(simulation)
 
     compute!(vorticity_computation)
 
     x, y = xnodes(ω)[:], ynodes(ω)[:]
-    heatmap(x, y, interior(ω)[:, :, 1]', xlabel="x", ylabel="y",
-            color=:balance, clims=(-0.1, 0.1))
+    heatmap(x, y, interior(ω)[:, :, 1]',
+            xlabel="x", ylabel="y",
+            aspectratio=1,
+            color=:balance,
+            clims=(-0.2, 0.2),
+            xlims=(0, model.grid.Lx),
+            ylims=(0, model.grid.Ly))
```

Comment:
@navidcy do you mind if I convert this example to run the simulation and post-process _afterwards_ (like in `langmuir_turbulence.jl`)? This is an outstanding issue #676

Reply:
I’ll do it as an exercise:)

Reply:
I’ll do it as an exercise:)